java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -872,6 +876,42 @@ public CompletableFuture<Long> zrangestore(
         return this.zrangestore(destination, source, rangeQuery, false);
     }
 
+    @Override
+    public CompletableFuture<String[]> zunion(
+            @NonNull WeightableKeys weightableKeys, @NonNull Aggregate aggregate) {
+        String[] arguments =
+                concatenateArrays(
+                        weightableKeys.toArgs(), new String[] {AGGREGATE_REDIS_API, aggregate.toString()});
```

Comment:
If you add `toArgs` method to `Aggregate` you can do
```suggestion
                        weightableKeys.toArgs(), aggregate.toArgs());
```

Reply:
Yes.  Better to keep the usage of AGGREGATE_REDIS_API within Aggregate.